<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HoughCircleForm</class>
 <widget class="QWidget" name="HoughCircleForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>999</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hough Circle Method</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="4" colspan="2">
    <widget class="QPushButton" name="buttonHoughCircle">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>3. Detect circles</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4" colspan="2">
    <widget class="QPushButton" name="buttonCanny">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>2. Show Canny filter</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4" colspan="2">
    <widget class="QPushButton" name="buttonOpenImage">
     <property name="text">
      <string>                    1. Open image                    </string>
     </property>
    </widget>
   </item>
   <item row="6" column="5">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="12" column="5">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="8" column="6">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>maxRadius</string>
     </property>
    </widget>
   </item>
   <item row="11" column="5">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Canny second parameter</string>
     </property>
    </widget>
   </item>
   <item row="13" column="5">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="buttonReset">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Reset values</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonSaveCanny">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Save Canny image as ...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonSaveHough">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Save Hough image as ...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="8" column="5">
    <widget class="QSlider" name="horizontalSliderParam1">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
     <property name="value">
      <number>250</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="9" column="5">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Canny first parameter</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>minRadius</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QSlider" name="verticalSliderMinRadius">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="maximum">
      <number>250</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="7" column="6">
    <widget class="QSlider" name="verticalSliderMaxRadius">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="maximum">
      <number>250</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>250</number>
     </property>
     <property name="tracking">
      <bool>true</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::NoTicks</enum>
     </property>
    </widget>
   </item>
   <item row="10" column="5">
    <widget class="QSlider" name="horizontalSliderParam2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
     <property name="value">
      <number>250</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="5">
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="7" column="5">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>256</width>
          <height>212</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="labelOriginalImage">
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QScrollArea" name="scrollArea_2">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>257</width>
          <height>212</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="labelCanny">
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QScrollArea" name="scrollArea_3">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>256</width>
          <height>212</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="labelHoughCircle">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="5">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Click to zoom in or out :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="5">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="pushButtonZoomOriginal">
       <property name="text">
        <string>Image</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonZoomCanny">
       <property name="text">
        <string>Canny</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonZoomHough">
       <property name="text">
        <string>Hough</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
